sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -569,3 +551,28 @@ def test_constant_target(kernel):
     assert_allclose(y_pred, y_constant)
     # set atol because we compare to zero
     assert_allclose(np.diag(y_cov), 0., atol=1e-9)
+
+
+def test_gpr_consistency_std_cov_non_invertible_kernel():
+    """Check the consistency between the returned std. dev. and the covariance.
+    Inconsistencies were observed when the kernel cannot be inverted (or
```

Comment:
```suggestion
    """Check the consistency between the returned std. dev. and the covariance.

    Inconsistencies were observed when the kernel cannot be inverted (or
```

Reply:
I changed formatting to be consistent with the preceding test